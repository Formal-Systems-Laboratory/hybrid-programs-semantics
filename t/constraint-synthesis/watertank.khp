params m, ep ; vars f, l, c ;f :=* ; ?(-1.0 <= f && f <= (m-l)/ep) ; c := 0.0 ;
l' = f, c' = 1.0 & l >= 0.0 , c <= ep
