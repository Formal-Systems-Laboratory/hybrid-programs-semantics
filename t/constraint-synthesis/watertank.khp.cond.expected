cpost >= 0 && -1.*ep <= 0. && ep > 0 && -1.*fpost <= 1. && -1.*l <= 0. && ep*fpost + l - m <= 0 && 1.*cpost - 1.*tpost == 0 && tpost >= 0 && -1.*ep + 1.*tpost <= 0 && -1.*l - 1.*fpost*tpost <= 0 && l - lpost + fpost*tpost == 0 && (-1.*fpost <= 0 || fpost == 0 || (l != 0 && -1.*fpost*l <= 0) || (1.*fpost*l <= 0 && 1.*fpost*l + 1.*fpost^2*tpost <= 0))
